node/trusted_validators_membership_change.re

```diff
@@ -0,0 +1,45 @@
+open Helpers;
+open Protocol;
+open Mirage_crypto_ec;
+
+module type T = {
```

Comment:
Why this module type here?

Reply:
At one point I was using first class modules somewhere, but figured a way without it. The type is irrelevant now and can be removed. Removing..